src/wallets/sui/index.ts

```diff
@@ -105,13 +110,12 @@ export class SuiWalletToolbox extends WalletToolbox {
   }
 
   public parseTokensConfig(tokens: string[], failOnInvalidTokens: boolean): string[] {
-    const knownTokens = this.getKnownTokens();
     const validTokens: string[] = [];
     for (const token of tokens) {
       if (this.isValidNativeTokenAddress(token)) {
         validTokens.push(token);
       } else if (this.isKnownToken(token)) {
```

Comment:
May be we can update the if condition as well:

```suggestion
     const knownTokens = this.getKnownTokens();
     ...
      } else if (token.toUpperCase() in knownTokens) {
        validTokens.push(knowTokens[token.toUpperCase()]);
      }
```

Reply:
Cool, looks good 👍